README.md

```diff
@@ -102,9 +104,9 @@ To achieve this, it is advised to allocate "idle" funds to any market on Morpho
 
 - The vault's asset as loan token.
 - No collateral token (`address(0)`).
-- An arbitrary IRM.
+- An IRM that does not trigger a revert (`address(0)`).
 - An arbitrary oracle (`address(0)`).
-- An arbitrary LLTV.
+- An arbitrary LLTV (`0`).
```

Comment:
Consider removing the values in parentheses and adding an example of suggested config to create a proper idle market. Explain properly which are the values that should be used and why it's important to pick those values. For example, you must set the collateral token to `address(0)` to prevent users from borrowing the supplied liquidity (and so on for each parameter)

Reply:
Hmm the values in parenthesis are the ones that should be set to create a canonic idle market. Do you have something else in mind?

Reply:
I think that being explicit with a proper example (explaining why you need to do it like that and specifying those specific values) would make things easy. Being explicit and restrictive is always better.

Also, explaining why to pick those values (and the unwanted side effects of not choosing them) adds value. For example, `address(0)` for the collateral is needed to prevent MB users to borrowing the idle liquidity.

I think that it's fair to think that people that will use MM are less deep into the knowledge and code as you (the owner and developer) are.